vue-frontend/src/pages/jobs/[id].vue

```diff
@@ -370,7 +374,7 @@ export default {
 
 .summary-text,
 :deep(div > span *),
-:deep(ul *) {
+:deep(ul) {
```

Comment:
Review description once, after changing these css.

Reply:
Description is similar to a live website then after change the css